test/sophia_bytecode_aevm_SUITE.erl

```diff
@@ -21,14 +21,20 @@ all() ->
       execute_identity_fun_from_sophia_file ].
 
 execute_identity_fun_from_sophia_file(_Cfg) ->
+    %% Should work for both versions of serialized contracts
+    execute_identity_fun_from_sophia_file(_Cfg, 1),
+    execute_identity_fun_from_sophia_file(_Cfg, 2).
```

Comment:
Is there a test checking that new format is invalid in old consensus? (For ensuring sanity of assumption in test.)

Reply:
Yes, I manually ran it against the contract_SUITE and basically all tests failed. I have considered adding a specific test for it, but thought it would polute the contract SUITE.

Reply:
I suggest you make a unit test in `aect_contract_create_tx_tests.erl` where you vary the code serialization in create transactions, and use
 `meck:expect(aec_hard_forks, protocol_effective_at_height, fun(_) -> ?MINERVA_PROTOCOL_VERSION end)`
and
`meck:expect(aec_hard_forks, protocol_effective_at_height, fun(_) -> ?ROMA_PROTOCOL_VERSION end)`
to see that the correct versions are accepted/rejected when calling
`aect_contract_create:process/3`

Reply:
Added a test in aecontract_SUITE now